www/kukd01/firma.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Tamadex</title>
+
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="main.css" />
+    <script src="main.js"></script>
+</head>
+
+<body>
+    <header>
+        <!--Proč se to nezarovnává na střed ? center nesmím (validátor radí css)-->
```

Comment:
tak zatím nejsou styly že?

Reply:
To mi tam dal automaticky VS Code při použití šablony :/ Odmazáno.

Reply:
no ja jsem myslel atribut class="centerImage" ted nema efekt protoze nejsou styly, ale v pohode, to upravis potom